src/main/java/seedu/address/logic/Messages.java

```diff
@@ -26,9 +26,9 @@ public class Messages {
                 "Oops! Multiple values are specified for the following single-valued parameter(s): ";
 
     public static final String MESSAGE_NON_INTEGER_INDEX = "Oops! The index you provided is not valid.\n"
-            + "It appears to be either not a valid integer or it exceeds the maximum allowed value "
-            + "(2147483647).\n"
-            + "Please try again with a valid positive integer!";
+            + "It should be a positive integer and within the range of 1 to "
+            + Integer.MAX_VALUE + ".\n"
+            + "Please try again with a valid index!";
 
     public static final String MESSAGE_NON_POSITIVE_INTEGER_INDEX = "Oops! The index you have given is not a "
```

Comment:
Are you sure this gives the correct output to the user, when the user types a number > Integer.MAX_VALUE?